relay-event-normalization/src/normalize/span/description/redis.rs

```diff
@@ -0,0 +1,95 @@
 LATEST|LATENCY HISTORY|LATENCY HISTOGRAM|LATENCY HELP|LATENCY GRAPH|LATENCY DOCTOR|LATENCY|LASTSAVE|KEYS|JSON.TYPE|JSON.TOGGLE|JSON.STRLEN|JSON.STRAPPEND|JSON.SET|JSON.RESP|JSON.OBJLEN|JSON.OBJKEYS|JSON.NUMMULTBY|JSON.NUMINCRBY|JSON.MSET|JSON.MGET|JSON.MERGE|JSON.GET|JSON.FORGET|JSON.DEL|JSON.DEBUG MEMORY|JSON.DEBUG HELP|JSON.DEBUG|JSON.CLEAR|JSON.ARRTRIM|JSON.ARRPOP|JSON.ARRLEN|JSON.ARRINSERT|JSON.ARRINDEX|JSON.ARRAPPEND|INFO|INCRBYFLOAT|INCRBY|INCR|HVALS|HTTL|HSTRLEN|HSETNX|HSET|HSCAN|HRANDFIELD|HPTTL|HPEXPIRETIME|HPEXPIREAT|HPEXPIRE|HPERSIST|HMSET|HMGET|HLEN|HKEYS|HINCRBYFLOAT|HINCRBY|HGETALL|HGET|HEXPIRETIME|HEXPIREAT|HEXPIRE|HEXISTS|HELLO|HDEL|GETSET|GETRANGE|GETEX|GETDEL|GETBIT|GET|GEOSEARCHSTORE|GEOSEARCH|GEORADIUS_RO|GEORADIUSBYMEMBER_RO|GEORADIUSBYMEMBER|GEORADIUS|GEOPOS|GEOHASH|GEODIST|GEOADD|FUNCTION STATS|FUNCTION RESTORE|FUNCTION LOAD|FUNCTION LIST|FUNCTION KILL|FUNCTION HELP|FUNCTION FLUSH|FUNCTION DUMP|FUNCTION DELETE|FUNCTION|FT._LIST|FT.TAGVALS|FT.SYNUPDATE|FT.SYNDUMP|FT.SUGLEN|FT.SUGGET|FT.SUGDEL|FT.SUGADD|FT.SPELLCHECK|FT.SEARCH|FT.PROFILE|FT.INFO|FT.EXPLAINCLI|FT.EXPLAIN|FT.DROPINDEX|FT.DICTDUMP|FT.DICTDEL|FT.DICTADD|FT.CURSOR READ|FT.CURSOR DEL|FT.CREATE|FT.CONFIG SET|FT.CONFIG HELP|FT.CONFIG GET|FT.ALTER|FT.ALIASUPDATE|FT.ALIASDEL|FT.ALIASADD|FT.AGGREGATE|FLUSHDB|FLUSHALL|FCALL_RO|FCALL|FAILOVER|EXPIRETIME|EXPIREAT|EXPIRE|EXISTS|EXEC|EVAL_RO|EVALSHA_RO|EVALSHA|EVAL|ECHO|DUMP|DISCARD|DEL|DECRBY|DECR|DEBUG|DBSIZE|COPY|CONFIG SET|CONFIG REWRITE|CONFIG RESETSTAT|CONFIG HELP|CONFIG GET|CONFIG|COMMAND LIST|COMMAND INFO|COMMAND HELP|COMMAND GETKEYSANDFLAGS|COMMAND GETKEYS|COMMAND DOCS|COMMAND COUNT|COMMAND|CMS.QUERY|CMS.MERGE|CMS.INITBYPROB|CMS.INITBYDIM|CMS.INFO|CMS.INCRBY|CLUSTER SLOTS|CLUSTER SLAVES|CLUSTER SHARDS|CLUSTER SETSLOT|CLUSTER SET-CONFIG-EPOCH|CLUSTER SAVECONFIG|CLUSTER RESET|CLUSTER REPLICATE|CLUSTER REPLICAS|CLUSTER NODES|CLUSTER MYSHARDID|CLUSTER MYID|CLUSTER MEET|CLUSTER LINKS|CLUSTER KEYSLOT|CLUSTER INFO|CLUSTER HELP|CLUSTER GETKEYSINSLOT|CLUSTER FORGET|CLUSTER FLUSHSLOTS|CLUSTER FAILOVER|CLUSTER DELSLOTSRANGE|CLUSTER DELSLOTS|CLUSTER COUNTKEYSINSLOT|CLUSTER COUNT-FAILURE-REPORTS|CLUSTER BUMPEPOCH|CLUSTER ADDSLOTSRANGE|CLUSTER ADDSLOTS|CLUSTER|CLIENT UNPAUSE|CLIENT UNBLOCK|CLIENT TRACKINGINFO|CLIENT TRACKING|CLIENT SETNAME|CLIENT SETINFO|CLIENT REPLY|CLIENT PAUSE|CLIENT NO-TOUCH|CLIENT NO-EVICT|CLIENT LIST|CLIENT KILL|CLIENT INFO|CLIENT ID|CLIENT HELP|CLIENT GETREDIR|CLIENT GETNAME|CLIENT CACHING|CLIENT|CF.SCANDUMP|CF.RESERVE|CF.MEXISTS|CF.LOADCHUNK|CF.INSERTNX|CF.INSERT|CF.INFO|CF.EXISTS|CF.DEL|CF.COUNT|CF.ADDNX|CF.ADD|BZPOPMIN|BZPOPMAX|BZMPOP|BRPOPLPUSH|BRPOP|BLPOP|BLMPOP|BLMOVE|BITPOS|BITOP|BITFIELD_RO|BITFIELD|BITCOUNT|BGSAVE|BGREWRITEAOF|BF.SCANDUMP|BF.RESERVE|BF.MEXISTS|BF.MADD|BF.LOADCHUNK|BF.INSERT|BF.INFO|BF.EXISTS|BF.CARD|BF.ADD|AUTH|ASKING|APPEND|ACL WHOAMI|ACL USERS|ACL SETUSER|ACL SAVE|ACL LOG|ACL LOAD|ACL LIST|ACL HELP|ACL GETUSER|ACL GENPASS|ACL DRYRUN|ACL DELUSER|ACL CAT|ACL)(?-u:\s|$)").unwrap()
```

Comment:
Should we still have the list of commands as a spearate `const` and just build the regex once with a loop?